source/includes/fact-mmapv1-big-endian.rst

```diff
@@ -1,5 +1,7 @@
 .. important::
 
-   MMAPv1 is not supported on big-endian architectures such as s390x.
-   MongoDB returns an error if you set MMAPv1 as the
-   storage engine on a big-endian system.
+   MMAPv1 is unsupported on big-endian architectures like the IBM
+   z/Architecture (``s390x``) and certain bi-endian architectures like
```

Comment:
I think this ultimately begs the question of what other bi-endian or little-endian archs we throw errors on (vs archs we simply don't test against for official support). If it's _only_ ppc64le that we throw errors for, we should just state that.

Reply:
Fixed but forgot to push.